src/docs/issue-platform.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: "Issue Platform"
+---
+
+The Issue Platform allows developers to create new issue types from arbitrary data sources.
+
+To use the Issue Platform you need to:
+1. <Link to="#writing-a-detector">Write a Detector</Link> 
+2. <Link to="#register-an-issue-type">Register an Issue Type</Link> 
+3. <Link to="#send-occurrences-to-the-issue-platform">Send Occurrences to the Issue Platform</Link> 
+4. <Link to="#Releasing-your-issue-type">Release</Link>
+
+## Writing a Detector
+The first step to introducing a new issue type to Sentry is to determine what type of issue you want to detect. It can be from any data stream, as long as you can produce an <Link to="#issue-occurrence-schema">Occurrence</Link> representing the issue
+
+## Register an Issue Type
+To register a new issue type you need to subclass `GroupType` https://github.com/getsentry/sentry/blob/ef76b0765f6c057af12b1cd0dc2de6b42a938c02/src/sentry/issues/grouptype.py#L98-L108
+
+When registering the class, set the following required attributes on the class:
+- `type_id`: A unique integer to identify your issue type.
+- `slug`: A unique slug to identify your issue type. Used to search by issue type in the issue stream
+- `description`: A human readable description of your issue type [include where this is used]
+- `category`: The overall category that your issue type belongs to. [include docs on creating a category]
+
+You may also set these optional attributes:
+- `noise_config`: A `NoiseConfig` that allows you to set policy on how many times a fingerprint must occur in a period of time before an issue is created
+
+Here's an example issue type:
+```python
+@dataclass(frozen=True)
+class ProfileFileIOGroupType(GroupType):
+    type_id = 2001
+    slug = "profile_file_io_main_thread"
+    description = "File I/O on Main Thread"
+    category = GroupCategory.PROFILE.value
```

Comment:
I think you should add an example of the `noise_config` in there.

Should you also document the `release` field for single tenant?

Reply:
Covered below in the release section, but it probably makes sense to link there from here.